performance/manifests/nats.yaml

```diff
@@ -0,0 +1,31 @@
+# https://github.com/nats-io/k8s/tree/main/helm/charts/nats
+config:
+  cluster:
+    enabled: true
+    replicas: 3
+  jetstream:
+    enabled: true
+    fileStore:
+      pvc:
+        size: 10Gi
+
+podTemplate:
+  topologySpreadConstraints:
+    kubernetes.io/hostname:
+      maxSkew: 1
+      whenUnsatisfiable: DoNotSchedule
```

Comment:
This block hard-guarantees that you will not have two NATS replicas on the same node, correct?  A short comment to that effect would be useful.